Reply/app/src/main/res/layout/activity_main.xml

```diff
@@ -38,64 +38,24 @@
-            app:hideOnScroll="true">
-
-            <LinearLayout
-                android:id="@+id/bottom_app_bar_content_container"
-                android:layout_width="wrap_content"
-                android:layout_height="@dimen/min_touch_target_size"
-                android:layout_marginVertical="@dimen/grid_0_5"
-                android:paddingHorizontal="@dimen/grid_0_5"
-                android:orientation="horizontal"
-                android:gravity="center_vertical"
-                android:foreground="@drawable/bottom_app_bar_title_foreground">
-
-                <ImageView
-                    android:id="@+id/bottom_app_bar_chevron"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:contentDescription="@string/bottom_app_bar_chevron_content_desc"
-                    android:layout_gravity="center_vertical"
-                    app:tint="?attr/colorOnPrimarySurface"
-                    app:srcCompat="@drawable/ic_arrow_drop_up" />
-
-                <ImageView
-                    android:id="@+id/bottom_app_bar_logo"
-                    android:layout_width="@dimen/bottom_app_bar_logo_size"
-                    android:layout_height="@dimen/bottom_app_bar_logo_size"
-                    android:layout_marginLeft="@dimen/grid_1"
-                    android:contentDescription="@string/bottom_app_bar_logo_content_desc"
-                    android:layout_gravity="center_vertical"
-                    app:srcCompat="@drawable/ic_reply_logo" />
-
-                <TextView
-                    android:id="@+id/bottom_app_bar_title"
-                    android:textAppearance="?attr/textAppearanceBodyLarge"
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    android:layout_gravity="center_vertical"
-                    android:layout_marginHorizontal="@dimen/grid_1"
-                    android:text="@string/navigation_inbox"
-                    android:textColor="?attr/colorControlNormal"/>
-
-            </LinearLayout>
-
-        </com.google.android.material.bottomappbar.BottomAppBar>
+            app:labelVisibilityMode="unlabeled"
+            app:menu="@menu/home_bottom_nav_menu" />
 
         <com.google.android.material.floatingactionbutton.FloatingActionButton
             android:id="@+id/fab"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            app:layout_anchor="@id/bottom_app_bar"
+            android:layout_marginEnd="@dimen/grid_2"
             android:contentDescription="@string/fab_compose_email_content_description"
+            app:fabCustomSize="@dimen/compose_email_fab_size"
+            app:layout_anchorGravity="end"
+            app:layout_insetEdge="bottom"
+            app:layout_anchor="@id/bottom_navigation"
             app:srcCompat="@drawable/asl_edit_reply" />
```

Comment:
Since the icon is no longer going to animate to a `reply` icon, this drawable can just be a static icon - `@drawable/ic_edit`

The drawable being used currently is an "animated state list" drawable that is able to morph between icons. There's a really cool tool that you can use to create these morphs if you're interested or if we need them later in the project - https://shapeshifter.design/